src/Uno.UI/UI/Xaml/Controls/ScrollBarVisibility.cs

```diff
@@ -10,14 +10,14 @@ namespace Windows.UI.Xaml.Controls
 	public enum ScrollBarVisibility
 	{
 		/// <summary>
-		/// Enables the scrollbars if the content is greater than the view port.
+		/// Disables scrolling.
 		/// </summary>
-		Auto,
+		Disabled,
```

Comment:
Given the order is set to intentionally give the same values as the UWP/WinUI enum with the same name, I'd suggest actually assigning the values to make it explicit. Also might be worth referencing the docs, so that someone doesn't change order - https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.scrollbarvisibility

Reply:
@nickrandolph The generator is now supposed to ensure correct values.

**All** enums should have the same values as Windows, and there are a lot. I think it's not worth adding a comment to every enum if we have an automated solution.